rest-api/src/main/java/uk/gov/hmcts/ccd/definition/store/rest/endpoint/CaseDefinitionController.java

```diff
@@ -61,6 +62,17 @@ public CaseType dataCaseTypeIdGet(
         return caseTypeService.findByCaseTypeId(id).orElseThrow(() -> new NotFoundException(id));
     }
 
+    @GetMapping(value = "/data/case-types", produces = {"application/json"})
+    @ApiOperation(value = "Get caseTypes", response = CaseType.class, responseContainer = "List")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "A Case Type Schema"),
+        @ApiResponse(code = 200, message = "Unexpected error")
```

Comment:
I know this is just a copy from the previous existing endpoint, but I doubt 200 for "Unexpected error" is correct so we should at least correct it for this new endpoint 😄

Reply:
hahahahaha :)

Reply:
DONE